spec/models/event_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe "Event model", type: :model do
+
+    it "should not validate without name" do
+        league = FactoryBot.build(:league)
+        league.name = nil
+
```

Comment:
you can do this: `league = FactoryBot,build(:league, name: nil)` if you want to specifically test if the constructor works.